app/controllers/boxes_controller.rb

```diff
@@ -53,6 +62,6 @@ def load_box
   def set_boxes_with_unread_message_counts(boxes)
     unread_messages_per_box = policy_scope(Message).joins(:thread).where(read: false, message_threads: { box_id: boxes.to_a }).group("message_threads.box_id").count
 
-    @boxes_with_unread_message_counts = boxes.map { |box| [box, unread_messages_per_box[box.id] || 0] }.to_h
+    @boxes_with_unread_message_counts = boxes.index_with { |box| unread_messages_per_box[box.id] || 0 }
```

Comment:
nice!

Reply:
Toto je zlate. Zial som v tom nevinne, to je praca mojho (tolko nenavideneho lintera). Ja ho mam pocas vyvoja v rezime Format on save, aby po mne neostaval bordel. Obcas teda ale uprace aj veci, ktore som nerobil ja, a casto ta to iritovalo, tak som to zacal po nom upratovavat. No a teraz spravil toto, a ja som si to nevsimol pri upratovani, tak to preklzlo ;-)

https://www.rubydoc.info/gems/rubocop-rails/2.5.2/RuboCop/Cop/Rails/IndexWith